mobile/src/Modal.js

```diff
@@ -98,7 +98,7 @@ export default class CustomModal extends Component {
               {this.props.showQuestion ? <TouchableOpacity style={s.circleBox}><Text style={s.whiteText}>?</Text></TouchableOpacity> : <Image style={s.pencilBox} source={pencil}/>}
               <TextInput style={Platform.select({ios: [newStyle, iosStyle], android: [newStyle, androidStyle]})} placeholder={this.props.showQuestion ? "What is your question?" : "Add your own answer"}
                 value={this.state.question}
-                onChangeText={question => this.setState({question})} 
+                onChangeText={question => this.updateQuestion(question)} 
```

Comment:
`onChangeText={this.updateQuestion}` is more performant, because it does not rebind a new function on every render.